environment.yml

```diff
@@ -21,3 +21,4 @@ dependencies:
   - trame-vuetify
   - 'nodejs>=20,<21'
   - mystmd
+  - imagemagick
```

Comment:
We are fixing this with a fall-back to `convert`. If you put this in because of the build logs complaining about `magick`, feel free to ignore those. cc @fwkoch.

Reply:
ah ok, cool - yeah, that was the reason!

Reply:
This is fixed here: https://github.com/jupyter-book/mystmd/pull/1410